### DISCUSSION TITLE
attu 访问 Protocol error

### DISCUSSION THREAD

**Discussion by walkerrbytes** [milvus-io/milvus#33501] [Q&A and General discussion] [answered]
attu容器指令
docker run -p 8000:3000 --name=attu -d -e HOST_URL=http://192.168.18.129:8000  -e MILVUS_URL=192.168.18.129:31786 zilliz/attu:v2.3.10
192.168.18.129为宿主机ip
<img width="530" alt="Snipaste_2024-05-31_11-48-13" src="https://github.com/milvus-io/milvus/assets/71864085/8a267906-b74e-4bd5-8852-92b06d106d8a">

访问attu页面报错，页面milvus地址为http://192.168.18.129:31786
Error: Failed to connect to Milvus: Error: 13 INTERNAL: Received RST_STREAM with code 2 triggered by internal client error: Protocol error

**Comment by yhmo** [comment, marked as answer]
MILVUS_URL=192.168.18.129:31786 你的milvus server可能用的不是31768这个端口

**Reply by walkerrbytes** [reply]
用错端口了，感谢回复
